app/src/main/java/org/cru/godtools/GodToolsApplication.kt

```diff
@@ -60,4 +65,14 @@ open class GodToolsApplication : Application() {
         }
         Timber.plant(CrashlyticsTree())
     }
+
+    // region WorkManager Configuration.Provider
+    init {
+        TimberLogger(Log.ERROR).install()
```

Comment:
I'm trying to figure out what this is doing exactly. I have a few ideas, but not sure which is right.

Reply:
the class name isn't the most helpful in this case. this `TimberLogger` is a WorkManager Logger that sends log messages to Timber. This is creating the logger, then installing it in WorkManager.

There isn't an official API to do this yet, which is the reason it's not in the work manager configuration.

Reply:
Ah okay, that makes sense. Thanks!